src/UI/Shared/FormatHelpers.js

```diff
@@ -3,16 +3,19 @@ var filesize = require('filesize');
 var UiSettings = require('./UiSettingsModel');
 
 module.exports = {
-    bytes : function(sourceSize) {
+    bytes : function(sourceSize, sourceRounding) {
         var size = Number(sourceSize);
-
+        var rounding = Number(sourceRounding||1);
```

Comment:
`undefined` will result in NaN, passing in `null` or `false` result in 0, but `undefined` will be the default. If someone passed in `0` we'd actually change the value to `1` with this.